lms/djangoapps/mobile_api/tests/test_milestones.py

```diff
@@ -136,4 +135,4 @@ def _verify_unfulfilled_milestone_response(self):
             self.api_response()
         else:
             response = self.api_response(expected_response_code=404)
-            self.assertEqual(response.data, MilestoneError().to_json())
+            self.assertEqual(response.data, {u'detail': u'Not found.'})
```

Comment:
Reviewers: I'm not sure this is the best idea, but looking through the code, fixing this is nontrivial.